css-canvas-medical/css/footer.css

```diff
@@ -0,0 +1,90 @@
+.learn-more-text {
+  padding-left: 13rem;
+  margin-bottom: 3rem;
+}
+
+.learn-more-title {
+  font-family: 'Tiempos Headline', serif;
+  font-size: 4rem;
+  font-weight: 100;
+  color: rgb(110, 110, 110);
+  margin-bottom: 0;
+}
+
+.learn-more-textbox {
+  font-family: Avenir, sans-serif;
+  font-size: 2rem;
+  color: rgb(126, 126, 126);
+}
+
+.learn-more-btn-container {
+  justify-content: end;
+  margin-bottom: 10rem;
+  margin-right: 15rem;
+}
+
+.learn-more-btn {
+  background-color: rgb(47, 67, 97);
+  color: rgb(255, 255, 255);
+  font-size: 2.5rem;
+  padding: 2rem 6rem;
+  border: 0 none;
+  border-radius: 0.5rem;
+  margin-top: 7rem;
+}
+
+.info {
+  background-color: rgb(82, 83, 81);
+  color: rgb(255, 255, 255);
+  font-family: Avenir, sans-serif;
+  padding-bottom: 10rem;
+}
+
+.info-container {
+  padding-left: 13rem;
+}
+
+.info-logo {
+  padding-top: 15rem;
+  height: 18.5rem;
+}
+
+.address {
+  font-size: 2rem;
+  line-height: 3.5rem;
+  margin-top: 5rem;
+}
+
+.info-container-2 {
+  margin-top: 13rem;
+  margin-left: 30rem;
```

Comment:
In an ideal world, you shouldn't have such high `margin` values. You usually need to do this as a result of something "hacky" to get it to appear in the correct place. Since we did this before we learned about responsiveness, I expect more of this hacky stuff, but as you move on throughout the program, keep in mind that you want to avoid stuff like this